CMakeLists.txt

```diff
@@ -223,49 +223,6 @@ IF(NOT(VMIME_BUILD_SHARED_LIBRARY OR VMIME_BUILD_STATIC_LIBRARY))
 	MESSAGE(FATAL_ERROR "You should select at least one library to build (either VMIME_BUILD_SHARED_LIBRARY or VMIME_BUILD_STATIC_LIBRARY must be set to YES.")
 ENDIF()
 
-# These next two lines are required but it is unclear exactly what they do.
```

Comment:
These should be OBE even with the old minimum version line
https://github.com/Kitware/CMake/commit/430cc2b4b7a817be2fbc445d6803163f1695a533